cmd/helm/list.go

```diff
@@ -124,6 +124,7 @@ func newListCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 	f.BoolVar(&client.Deployed, "deployed", false, "show deployed releases. If no other is specified, this will be automatically enabled")
 	f.BoolVar(&client.Failed, "failed", false, "show failed releases")
 	f.BoolVar(&client.Pending, "pending", false, "show pending releases")
+	f.StringVarP(&client.ServiceName, "servicename", "s", "", "Name of the helm deployment")
```

Comment:
Service and Deployment are names used by Kubernetes. In this case I think you mean Release. The client property, flag name, and description should reflect that.

Reply:
I'd also ask how this would interact with other similar flags like `--filter` and `--selector`.

Reply:
The current `--filter` is a custom regex filter after helm list all the secrets. For large deployments helm usually timeout before the filter is applied.